gocourse02/faker.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"github.com/brianvoe/gofakeit/v6" // fake data
+)
+
+// Create structs with random injected data
+type Fake struct {
+	Str     string
+	Int     int
+	Pointer *int
+	Name    string `fake:"{firstname}"` // Any available function all lowercase
+	Gender  string `fake:"{gender}"`
+}
+
+type Bar struct {
+	Name   string
+	Number int
+	Float  float32
+}
+
+func GetFaker() Fake {
```

Comment:
Може краще перейменувати? Це ж типу конструктор:
```suggestion
func NewFake() Fake {
```

Ще можна додати коментар:

```go
// NewFakes returns Fake filled with random data.
func NewFake() Fake {
```